### PR TITLE
Split audit tool

### PR DESCRIPTION
**Pull request by chao-xian**
We are splitting out the Content Audit Tool from the Content Performance Manager. We are effectively freezing development on the Audit Tool, but it is in beta with users at the moment and needs to be available.

However we are going to be developing the Content Performance Manager extensively and as the Audit Tool is so coupled it is a risk to keep the two together - changes CPM code may break the Audit Tool and becomes a maintenance headache. This is the best, cleanest option at present.

### CODE REVIEW

File: hieradata/production.yaml [hunk 1]
```diff
@@ -68,6 +68,8 @@ base::supported_kernel::enabled: false
 
 environment_ip_prefix: '10.3'
 
+govuk::apps::content_audit_tool::ensure: 'absent'
```
Comment (cbaines) [inline hieradata/production.yaml]: Are you planning to change this to present at some point?

Comment (chao-xian) [reply hieradata/production.yaml]: @cbaines yep. We need to coordinate with a dept on when we can do the switchover. I thought it was a good idea to use this flag for now and not worry about the production deployment just yet.

Comment (cbaines) [reply hieradata/production.yaml]: Ok, that sounds fine :)

File: modules/govuk/manifests/apps/content_audit_tool.pp [hunk 2]
```diff
@@ -0,0 +1,157 @@
+# == Class: govuk::apps::content_audit_tool
+#
+# App details at: https://github.com/alphagov/content-audit-tool
+#
+# === Parameters
+#
+# [*ensure*]
+# Whether the application should be exist. Can be specified for each
+# environment using deployment hieradata.
```
Comment (cbaines) [inline modules/govuk/manifests/apps/content_audit_tool.pp]: There is some inconsistent indentation here.

File: modules/govuk/manifests/apps/content_audit_tool/db.pp [hunk 3]
```diff
@@ -0,0 +1,23 @@
+# == Class: govuk::apps::content_audit_tool::db
+#
+# === Parameters
+#
+# [*password*]
+#   The DB instance password.
+#
+# [*backend_ip_range*]
+#   Backend IP addresses to allow access to the database.
+#
```
Comment (cbaines) [inline modules/govuk/manifests/apps/content_audit_tool/db.pp]: The rds parameter is missing from the list in the comment.

Comment (chao-xian) [reply modules/govuk/manifests/apps/content_audit_tool/db.pp]: Addressed in https://github.com/alphagov/govuk-puppet/pull/7141/commits/9c4dedacec13d8f0f2fbdf3dd1f0620a84e4c920

**Pull request merged**